spring-cloud-netflix-zuul/src/main/java/org/springframework/cloud/netflix/zuul/filters/post/SendErrorFilter.java

```diff
@@ -100,10 +103,18 @@ public Object run() {
 
 	protected ExceptionHolder findZuulException(Throwable throwable) {
 		if (throwable.getCause() instanceof ZuulRuntimeException) {
+			Throwable cause = throwable.getCause().getCause().getCause();
```

Comment:
I am not to fond of this type of code as it could result in a NPE, I would want some checks here to protect us.